src/screens/Recovery/Lightning.tsx

```diff
@@ -0,0 +1,297 @@
+			selectedNetwork,
+		);
+
+		if (res.isErr()) {
+			console.log(res.error);
+			setIsFetchingBackup(false);
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: res.error.message,
+			});
+			return;
+		}
+
+		const bytesToStringRes = bytesToString(res.value.content);
+		if (bytesToStringRes.isErr()) {
+			console.log(bytesToStringRes.error);
+			setIsFetchingBackup(false);
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: bytesToStringRes.error.message,
+			});
+			return;
+		}
+
+		setBackup(JSON.parse(bytesToStringRes.value));
+		setIsFetchingBackup(false);
+		setShowConfirmRecoveryDialog(true);
+	};
+
+	const onShowLdkRecoveryConfirmed = async (): Promise<void> => {
+		if (!backup) {
+			return;
+		}
+
+		if (Object.keys(backup.data.channel_monitors).length === 0) {
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: t('lightning_recovery_no_channels'),
+			});
+			return;
+		}
+
+		setShowConfirmRecoveryDialog(false);
+		setIsRecoveringChannels(true);
+
+		await ldk.stop();
+
+		const storageRes = await setLdkStoragePath();
+		if (storageRes.isErr()) {
+			console.error(storageRes.error);
+			setIsRecoveringChannels(false);
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: storageRes.error.message,
+			});
+			return;
+		}
+
+		const importRes = await lm.importAccount({
+			backup,
+		});
+		if (importRes.isErr()) {
+			console.error(importRes.error);
+			setIsRecoveringChannels(false);
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: importRes.error.message,
+			});
+			return;
+		}
+
+		const setupRes = await startWalletServices({
+			onchain: false,
+			lightning: true,
+			restore: false,
+			staleBackupRecoveryMode: true,
+		});
+
+		if (setupRes.isErr()) {
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: setupRes.error.message,
+			});
+			setIsRecoveringChannels(false);
+			return;
+		}
+
+		const balances = await ldk.claimableBalances(false);
+		if (balances.isErr()) {
+			showToast({
+				type: 'error',
+				title: t('lightning_recovery_error'),
+				description: balances.error.message,
+			});
+			setIsRecoveringChannels(false);
+			return;
+		}
+
+		await ldk.stop();
+
+		let sats = 0;
+		balances.value.forEach((balance) => {
+			sats += balance.claimable_amount_satoshis;
+		});
+
+		setRecoveredSats(sats);
+		setShowLdkRecoverySuccessDialog(true);
+		setIsRecoveringChannels(false);
+	};
+
+	const onCloseApp = (): void => {
+		RNExitApp.exitApp();
+	};
+
+	return (
+		<ThemedView style={styles.root}>
+			<SafeAreaInset type="top" />
+			<NavigationHeader title={t('lightning_recovery_title')} />
+			<View style={styles.content}>
+				{isRecoveringChannels ? (
+					<ActivityIndicator color="white" />
+				) : (
+					<List data={[history]} />
+				)}
```

Comment:
Added an empty state: 

```suggestion
				{isRecoveringChannels ? (
					<ActivityIndicator color="white" />
				) : (
					<>
						{history.data.length === 0 ? (
							<Text01S>No backups found.</Text01S>
						) : (
							<List data={[history]} />
						)}
					</>
				)}
```